src/StructuredLogger/PlatformUtilities.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace StructuredLogger;
+
+public class PlatformUtilities
+{
+    public static bool HasThreads => !RuntimeInformation.IsOSPlatform(_browser);

```

Comment:
Is the function IsOSPlatform fast?  Can we cache the result since the value won't change?

Reply:
Done